src/test/javascript/spec/component/text-submission-assessment/textblock-feedback-editor.component.spec.ts

```diff
@@ -209,6 +227,55 @@ describe('TextblockFeedbackEditorComponent', () => {
         });
     });
 
+    it('should connect automatic feedback origin blocks with current feedback', fakeAsync(() => {
+        component.feedback.suggestedFeedbackOriginSubmissionReference = 1;
+        component.feedback.suggestedFeedbackParticipationReference = 1;
+        const textAssessmentService = TestBed.inject(TextAssessmentService);
+
+        const participation: StudentParticipation = {
+            type: ParticipationType.STUDENT,
+        } as unknown as StudentParticipation;
+
+        const textSubmission = {
+            submissionExerciseType: SubmissionExerciseType.TEXT,
+            id: 1,
+            submissionDate: dayjs('2019-07-09T10:47:33.244Z'),
+            text: 'First text. Second text.',
+            participation,
+        } as unknown as TextSubmission;
+
+        textSubmission.results = [
+            {
+                id: 2374,
+                completionDate: dayjs('2019-07-09T11:51:23.251Z'),
+                textSubmission,
+            } as unknown as Result,
+        ];
+        textSubmission.latestResult = getLatestSubmissionResult(textSubmission);
+        textSubmission.latestResult!.feedbacks = [
+            {
+                id: 1,
+                reference: 'First text id',
+            } as Feedback,
+        ];
+        textSubmission.blocks = [
+            {
+                id: 'First text id',
+                text: 'First text.',
+                textSubmission,
+            } as unknown as TextBlock,
+        ];
+        participation.submissions = [textSubmission];
+
+        const participationStub = jest.spyOn(textAssessmentService, 'getFeedbackDataForExerciseSubmission').mockReturnValue(of(participation));
+
+        component.connectAutomaticFeedbackOriginBlocksWithFeedback();
+        tick();
+
+        expect(participationStub).toHaveBeenCalledTimes(1);
+        expect(component.listOfBlocksWithFeedback).not.toBeUndefined();
```

Comment:
Could we expect a real value here?
```suggestion
        expect(component.listOfBlocksWithFeedback).toEqual([...]);
```